### ISSUE TITLE
Relative urls in css url() function not working in playground

### ISSUE THREAD

**Issue by bennlich** [jhpyle/docassemble#154]
Copied relevant pieces from https://github.com/jhpyle/docassemble/issues/153.

In the [playground](https://mycitations.uat.courts.ca.gov/interview?i=docassemble.playground1%3Aa2p-entrypoint.yml#page1):
```
/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png
```
resolves to
```
https://abilitytopayblobtest.blob.core.windows.net/abilitytopayblobtestcontainer/playgroundstatic/1/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png
```

In [normal-interview-land](https://mycitations.uat.courts.ca.gov/interview?i=docassemble.jcc.abilitytopay%3Adata%2Fquestions%2Finterview.yml#page1), the same absolute path resolves to:
```
https://mycitations.uat.courts.ca.gov/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png
```

The problem I'm having is that the urls beginning with https://abilitytopayblobtest.blob.core.windows.net/ do not succeed.

E.g. https://abilitytopayblobtest.blob.core.windows.net/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png

So the images that show up in my normal interviews do not show up in my playground tests.

**Comment by jhpyle** [comment]
Could you be more specific?  A copy and paste from your code and the text of the error message would be helpful.  I don't know what you mean by `/packagestatic/...` "resolves to" a URL.  An HTTP request to the `/packagestatic` endpoint will not respond with a 302 redirect unless the package name is `fonts`.

The URLs to windows.net are supposed to have a `?rscd=...` parameter after them, which contains the necessary access code.  These are temporary URLs that expire after an hour.  The Blob Storage container should be "Private" but the temporary URLs provide access when the server specifically grants access.  I did a test on a site that uses Azure Blob Storage, and as far as I can tell, the `?rscd=...` parameter is successfully being added when a `[FILE ...]` reference is made.  I can see why HTTP requests to the URLs would fail if they don't have `?rscd=...` added to them.  Another reason they might fail is if you have set restrictive settings on the Blob Storage container that blocks access to the container's files from the outside world, even if an access code is provided.

I discovered a problem with `static_image()` where it only works with file references that contain a package name.  It was substituting `docassemble.base` as the package for unqualified image names.  This will be fixed in the next version.

**Comment by bennlich** [comment]
>  I don't know what you mean by /packagestatic/... "resolves to" a URL.

By "resolves to" I mean, what url does the browser actually make an HTTP request to when given an absolute path (in a src attribute or elsewhere).

I did a little more digging, and found the issue. I have an element with this style:
```
background-image: url(/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png);
```
This appears to cause the browser to hit https://abilitytopayblobtest.blob.core.windows.net/packagestatic/docassemble.jcc.abilitytopay/california-courts-logo.png with no `?rscd` param, so the request fails with a 404.

Looking at the [url()](https://developer.mozilla.org/en-US/docs/Web/CSS/url) docs, this path is selected because relative urls are treated as relative to the stylesheet url.

In the playground, the stylesheet url is https://abilitytopayblobtest.blob.core.windows.net/abilitytopayblobtestcontainer/playgroundstatic/1/style.css?sp=r&rscd=, and in normal-interview-land, the stylesheet url is https://mycitations.uat.courts.ca.gov/packagestatic/docassemble.jcc.abilitytopay/style.css. So this explains why the resource loads in normal-interview-land but not in the playground.

Can I fix this? Any idea why my stylesheet is being served from a different location in the playground than in normal-interview-land? In both cases I'm referencing the stylesheet in my interview like so:
```
---
features:
  bootstrap theme: style.css
---
```

**Comment by jhpyle** [comment]
The interview should work from the Playground if you do:
```
features:
  bootstrap theme: https://mycitations.uat.courts.ca.gov/packagestatic/docassemble.playground1/style.css
```
When S3/Azure Blob Storage is used, static files in the Playground are served using temporary URLs so that the file can be transferred directly from cloud storage (where it lives) to the browser, rather than making two hops (cloud to server, server to browser).  The only reason for this was improving file transfer speed.  But I can see how that will interfere with relative URLs in CSS files, so I'll change it so that files are served from the server.  That way you can write `bootstrap theme: style.css` and `url(california-courts-logo.png)` and never have to change it when you go between development and production.

**Comment by jhpyle** [comment]
I believe I fixed this in 0.4.44 but feel free to reopen if the issue persists.
